scipy/stats/_distribution_infrastructure.py

```diff
@@ -0,0 +1,5082 @@
+
+        Evaluate the kurtosis:
+
+        >>> X.kurtosis()
+        3.0
+        >>> (X.kurtosis()
+        ...  == X.kurtosis(convention='excess') + 3.
+        ...  == X.moment(order=4, kind='standardized'))
+        True
+
+        """
+        non_excess = {'non-excess', 'pearson'}
+        excess = {'excess', 'fisher'}
+        message = (f'Parameter `convention` of `{self.__class__.__name__}.kurtosis` '
+                   "must be one of {'non-excess', 'Pearson', 'excess', 'Fisher'}.")
+        convention = convention.lower()
+        if convention not in non_excess.union(excess):
+            raise ValueError(message)
+        k = self.moment(4, kind='standardized', method=method)
+        return k - 3 if convention in excess else k
+
+    ### Distribution functions
+    # The following functions related to the distribution PDF and CDF are
+    # exposed via a public method that accepts one positional argument - the
+    # quantile - and keyword options (but not distribution parameters).
+    # logpdf, pdf
+    # logcdf, cdf
+    # logccdf, ccdf
+    # The `logcdf` and `cdf` functions can also be called with two positional
+    # arguments - lower and upper quantiles - and they return the probability
+    # mass (integral of the PDF) between them. The 2-arg versions of `logccdf`
+    # and `ccdf` return the complement of this quantity.
+    # All the (1-arg) cumulative distribution functions have inverse
+    # functions, which accept one positional argument - the percentile.
+    # ilogcdf, icdf
+    # ilogccdf, iccdf
+    # Common keyword options include:
+    # method - a string that indicates which method should be used to compute
+    #          the quantity (e.g. a formula or numerical integration).
+    # Tolerance options should be added.
+    # Input/output validation is provided by the `_set_invalid_nan`
+    # decorator. These are the methods meant to be called by users.
+    #
+    # Each public method calls a private "dispatch" method that
+    # determines which "method" (strategy for calculating the desired quantity)
+    # to use by default and, via the `@_dispatch` decorator, calls the
+    # method and computes the result.
+    # Each dispatch method can designate the responsibility of computing
+    # the required value to any of several "implementation" methods. These
+    # methods accept only `**kwargs`, the parameter dictionary passed from
+    # the public method via the dispatch method.
+    # See the note corresponding with the "Distribution Parameters" for more
+    # information.
+
+    ## Probability Density Functions
+
+    @_set_invalid_nan
+    def logpdf(self, x, *, method=None):
+        r"""Log of the probability density function
+
+        For more information about the probability density function,
+        see `ContinuousDistribution.pdf`.
+
+        Parameters
+        ----------
+        x : Array
+            The value of the random variable for which the log-PDF is to be
```

Comment:
drop "of the random variable"?

Reply:
Merging discussion with https://github.com/mdhaber/scipy/pull/110#discussion_r1554845190. Let's avoid duplication in future reviews.